slyd/app/controllers/spider.js

```diff
@@ -397,7 +389,7 @@ export default BaseController.extend({
     autoFetch: function() {
         if (this.get('loadedPageFp') && this.get('showLinks')) {
             this.saveSpider().then(function() {
-                this.fetchPage(this.get('pageMap')[this.get('loadedPageFp')].url, null, true);
+                //this.fetchPage(this.get('pageMap')[this.get('loadedPageFp')].url, null, true);
```

Comment:
Remove comment, is this function still needed? As far as I can see, `saveSpider` will be called anyway on any change to the model in `attachAutoSave`